### DISCUSSION TITLE
Adding Qtpy graphics view to flet container

### DISCUSSION THREAD

**Discussion by guillermollopis** [flet-dev/flet#1373] [Q&A]
### Question

I wonder if it is possible to add to a flet container an object of a different UI package. For example, I would like to add content created with qtpy. But flet containers just accept layout content. 

Does anyone know if this is possible or if there is another way of doing it?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
I personally don't think this is possible or could be even possible someday. Flet's builds up Flutter widgets, which I think can't move along with python qtpy widgets/components.
Flet controls only accept flet controls - at least for the moment :)
